specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -536,6 +549,7 @@
         "Consecutive",
         "Total"
       ],
+      "default" : "Consecutive",
```

Comment:
This is a fix to s360 violation :
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=D6E5FC34-0663-472E-952E-8B9F5C0C8D17
When "MetricTriggerType" is not specified by PUT, the service populates it with the default value of 'Consecutive'.
Therefor, this is not a breaking (we do not change the service current behavior) change but an alignment of the swagger.